README.md

```diff
@@ -5,45 +5,45 @@ This is the source code of ABRT's blog. You can visit it on
 # How to write a new article
 1. Write the article
 2. Create a new file in the `/_post` directory that follows the convention `YYYY-MM-DD-name-of-post.markdown`
-3. The file consists of two part devided by `---`
+3. The file consists of two parts separated by `---`
 
     ```
     ---
     layout: post
     title:  "Title of your post"
     date:   2017-01-22 18:48:44 +0100
-    author: your_name
+    author: <Your Name>
     visible: 1
-    categories: devided by space
+    categories: <space-separated list>
     ---
-    Here put your article in markdown
+    Put your article in markdown format here.
```

Comment:
If we want to be really pedantic, Markdown is spelled with the initial letter in uppercase.

Reply:
Thanks, fixed.

Reply:
Do we ever not want to be pedantic?